src/concepts/agents.md

```diff
@@ -24,17 +24,21 @@ Agents that run concurrently use [web-workers](https://developer.mozilla.org/en-
 * Public - Same as Context, but runs on its own web worker.
 * Global \(WIP\)
 
-## Agent Communication
+## Communication between Agents and Components
+
 
 ### Bridges
 
-Bridges will connect to an agent and allow two way communication.
+A bridge allows bi-directional communication between an agent and a component.
```

Comment:
Actually, agents can also communicate with each other. Could you update this reflect that?

Reply:
I've updated it to state this as well.